### ISSUE TITLE
loss function is not the same with the paper.In your code, you div the batch_size, but in the papr div N(the number of matched default boxes)?could you give me some advises?

### ISSUE THREAD

**Issue by fanbinqi** [balancap/SSD-Tensorflow#292]
loss function is not the same with the paper.In your code, you div the batch_size, but in the papr div N(the number of matched default boxes)?

**Comment by Hizhaoyuan** [comment]
hi, do you have some ideas about this issue?

**Comment by yangdi92** [comment]
@fanbinqi i have same ques? do you know the code of loss function in caffe?
